Upstream-Status: Backport
Signed-off-by: Ross Burton <ross.burton@intel.com>

From 622004565eca385c685086cd478aa79afe73b785 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Tue, 11 Jun 2013 17:16:37 +0100
Subject: [PATCH] build-sys: don't install quotaon.service twice

quotaon.service is already installed through dist_systemunit_DATA, so it doesn't
need to be added to nodist_systemunit_DATA.  Installing the same file twice
results in a race condition where the install process can fail.

https://bugs.freedesktop.org/show_bug.cgi?id=65659

[zj: actually remove quotaon.service from the other list.]
---
 Makefile.am |    1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 3a196a6..d444eac 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -401,7 +401,6 @@ dist_systemunit_DATA = \
 	units/bluetooth.target \
 	units/smartcard.target \
 	units/systemd-tmpfiles-clean.timer \
-	units/quotaon.service \
 	units/systemd-ask-password-wall.path \
 	units/systemd-ask-password-console.path \
 	units/systemd-udevd-control.socket \
-- 
1.7.10.4

